khi_robot_control/package.xml

```diff
@@ -34,6 +34,11 @@
   <run_depend>trajectory_msgs</run_depend>
   <run_depend>transmission_interface</run_depend>
   <run_depend>khi_robot_msgs</run_depend>
+  <run_depend>khi_duaro_moveit_config</run_depend>
+  <run_depend>khi_rs007n_moveit_config</run_depend>
+  <run_depend>khi_rs007l_moveit_config</run_depend>
+  <run_depend>khi_rs080n_moveit_config</run_depend>
 
   <test_depend>rostest</test_depend>
+  <test_depend>moveit_commander</test_depend>
```

Comment:
And the same comment here of course, although it is just a `test_depend`.

Reply:
@gavanderhoorn  `moveit_commander` is needed for testing the functions of `ros_control` and `joint_limits_interface`. It would be not needed to remove, I think.

Reply:
MoveIt Commander will drag in the rest of MoveIt, just as depending on MoveIt configuration packages does.

It is a `test_depend` here though, so that's slightly less 'bad' (in that they can be ignored if not running tests).